alerts/openshift-virtualization-operator/WaspNamespaceDetected.md

```diff
@@ -0,0 +1,45 @@
+# WaspNamespaceDetected
+
+## Meaning
+
+This alert fires when there is a duplicate wasp-agent deployment on the cluster
+
+## Impact
+Inconsistent behavior of workload swap configuration.
+
+## Diagnosis
+Identify the duplicate wasp-agent namespace:
+   ```bash
+   oc get ns wasp
+   ```
+
+## Mitigation
+
+1. Delete the wasp-agent Daemonset
+   ```bash
+   oc delete ds wasp-agent -n wasp
+   ```
+2. Delete the wasp-agent service account
+   ```bash
+   oc delete sa wasp -n wasp
+   ```
+3. Delete the wasp-agent cluster role binding
+   ```bash
+   oc delete clusterrolebinding wasp
+   ```
+4. Remove the `wasp` service account from privileged SCC
+   ```bash
+   oc adm policy remove-scc-from-user -n wasp privileged -z wasp
+   ```
+
+## Additional notes
+
+<!--DS: If you cannot resolve the issue, log in to the
```

Comment:
not sour about this DS comment 

cc @sradco

Reply:
Since is a downstream only runbook, we need here the DS text only